CHANGELOG.md

```diff
@@ -1,3 +1,95 @@
+## Release 0.17.0
+
+Highlights:
+- New Grid view as an alternative to Scopes classic graph view.  It provides
+  higher information density, proving particularly useful when there's a high
+  node count in the graph view.
```

Comment:
"there's a high node count" is pretty inside baseball. "there are many nodes" might be more friendly.